README.md

```diff
@@ -28,6 +28,38 @@ that shouldn't be difficult to implement (patches welcome).
 ### Installation
 Alpha release is available on Pypi - https://pypi.org/project/wagtail-markdown/ - installable via `pip install wagtail-markdown`. It's not a production ready release.
 
+The SimpleMDE editor uses FontAwesome for its widget icons. You need to add the
```

Comment:
would it also work with [wagtailfontawesome](https://gitlab.com/alexgleason/wagtailfontawesome)?

Reply:
ah, yes! I keep forgetting about it

Reply:
added note with link and installation instructions